cli.js

```diff
@@ -7,6 +7,7 @@ const { translate } = require('./utils/translate');
 const { configPath } = require('./utils/getConfigPath');
 const { getOrCreateTranslationFile } = require('./utils/getTranslationFile');
 const fs = require('fs');
+const { dset: set } = require('dset');
```

Comment:
I feel this name ("set") should not be used to avoid confusion in future contributors. Maybe use `dset` or something more descriptive like `setDeepValue`

Reply:
Hi! Thanks for the suggestion. setDeepValue sounds good to me. Already changed the name, please take a look when u can!